release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3400,6 +3400,41 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.14.52
+[id="ocp-4-14-52_{context}"]
+=== RHSA-2025:7702 - {product-title} 4.14.52 bug fix and security update
+
+Issued: 21 May 2025
+
+{product-title} release 4.14.52, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:7702[RHSA-2025:7702] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:7704[RHBA-2025:7704] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.52 --pullspecs
+----
+
+[id="ocp-4-14-52-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, an operator would incorrectly appear twice in the installed operators list if the current project matched its default namespace and the copied CSVs were disabled in {olm-first}. With this release, the operator displays only once. (link:https://issues.redhat.com/browse/OCPBUGS-55942[OCPBUGS-55942])
```

Comment:
```suggestion
* Previously, an operator would incorrectly appear twice in the installed operators list if the current project matched its default namespace and the copied CSVs were disabled in {olmv1-first}. With this release, the operator displays only once. (link:https://issues.redhat.com/browse/OCPBUGS-55942[OCPBUGS-55942])
```
That attribute doesn't exist in the 4.14 branch; @michaelryanpeter can we use the v1.0 attribute in this case, or should we just say "Operator Lifecycle Manager" and call it done?

Reply:
Correct. I would switch to "Operator Lifecycle Manager." Also, Operator needs to be capitalized throughout.